catroid/src/main/java/org/catrobat/catroid/devices/arduino/ArduinoListener.java

```diff
@@ -45,6 +45,7 @@
 	private int analogPin5 = 0;
 
 	int[] portValue = new int[]{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+	int[] uCPortValue = new int[] { 0, 0 };
```

Comment:
I renamed portValue (and associated getter&setter) to pinValue as this reflects its actual usage. I then renamed uCPortValue to portValue (the name that probably fits best and thatI wanted to use initially, but was already taken).